protocol-units/da/m1/light-node-verifier/src/celestia/mod.rs

```diff
@@ -1,79 +1,70 @@
-use crate::Verifier;
-use celestia_rpc::{BlobClient, Client, HeaderClient};
+use crate::{Error, Verified, VerifierOperations};
+use celestia_rpc::Client;
+#[cfg(feature = "pessimistic")]
+use celestia_rpc::{BlobClient, HeaderClient};
 use celestia_types::{nmt::Namespace, Blob};
-use m1_da_light_node_grpc::VerificationMode;
+use m1_da_light_node_util::inner_blob::InnerBlob;
```

Comment:
`InnerBlob` is an interesting name for public API...

Reply:
Yeah, I wasn't sure what we would best do here. We already have `Blob` from gRPC and `Blob` from Celestia. My thinking here was that this would be a consistent intermediate type that would be leveraged regardless of the underlying DA.

Reply:
Or "transaction payload", but we need to name it consistently throughout then.